libraries/abstractions/ble_hal/test/include/iot_test_ble_hal_common.h

```diff
@@ -226,7 +227,8 @@ typedef enum
     eBLEHALEventSSPrequestConfirmationCb = 22,
     eBLEHALEventPairingStateChangedCb = 23,
     eBLEHALEventRequestExecWriteCb = 24,
-    eBLENbHALEvents,
+    eBLENbHALEvents = 25,
```

Comment:
eBLENbHALEvents  denotes max HAL events so it should be after eBLEHALEventMtuChangedCb

Reply:
ok